queries.md

```diff
@@ -2,16 +2,28 @@
 
 ## find all customers that live in London. Returns 6 records.
 
+select \* from customers where city = 'London'
+
 ## find all customers with postal code 1010. Returns 3 customers.
 
+select \* from customers where postalcode = '1010'
+
 ## find the phone number for the supplier with the id 11. Should be (010) 9984510.
 
+select \* from suppliers where supplierid = 11
```

Comment:
Here the instruction says you need to find the phone number and you are searching instead for all. needs to be fixed.

Reply:
made changes-
select phone from suppliers where supplierId = 11